src/main/kotlin/no/nb/mlt/wls/core/data/Packaging.kt

```diff
@@ -0,0 +1,8 @@
+package no.nb.mlt.wls.core.data
+
+enum class Packaging {
+    OBJ,
+    ESK,
+    ABOX,
+    NONE
```

Comment:
Need to decide if this should be also a DTO, since in SynQ and API we use different terms. So we should probably have enums for both DTOs and internal logic.
In API we have BOX, ABOX, CRATE, and NONE. 
In SynQ we have ESK, ABOX, EA (Each, I think that one's not actually used in production), and OBJ

Reply:
Added a PackagingDTO for this purpose